pkg/config/globalcp/globalcp.go

```diff
@@ -0,0 +1,24 @@
+package globalcp
+
+import "github.com/Kong/kuma/pkg/config"
+
+var _ config.Config = &GlobalCPConfig{}
+
+// Global CP configuration
+type GlobalCPConfig struct {
+	// Registered Loca CPs for this Global CP
+	LocalCPs map[string]string `yaml:"localCPs,omitempty"`
```

Comment:
The key is the cluster name, and the value is the URL to access it for the sync procedure.

Reply:
What would be the better way to describe it?

Reply:
Just specify in the comment that key is cluster name and value is URL.

Btw. URL is for which endpoint?

Reply:
URL is "something" that can attest that the Local CP is alive. In this implementation, I use the API 5681 of Local CP to poll for "licenses". This can and probably will be changed once we have real XDS/communication.

Reply:
Should we provide the name here? I think that cluster name should be specified in local CP, so here you need list of local cps with address for sync server and ingress. Just like described in proposal https://github.com/Kong/kuma/blob/master/docs/proposals/multicluster.md#kuma-local-cp-registration

status endpoint can (and should) expose cluster names but I don't think config should contain this.